training_mod_consts/src/lib.rs

```diff
@@ -642,6 +644,14 @@ pub unsafe fn ui_menu(menu: TrainingModpackMenu) -> UiMenu {
         true,
         &(menu.crouch as u32),
     );
+    defensive_tab.add_submenu_with_toggles::<OnOff>(
+        "Dodge Staling".to_string(),
+        "stale_dodges".to_string(),
+        "Dodge Staling: Controls whether the CPU's dodges will worsen with repetitive use"
```

Comment:
Maybe newline here + information about how this can create scenarios that are not possible in-game